src/analysis/refiner.jl

```diff
@@ -49,7 +49,9 @@ Compiler.cache_owner(::StructuralRefiner) = StructureCache()
     end
 
     callee_codeinst = invokee
-    callee_result = structural_analysis!(callee_codeinst, Compiler.get_inference_world(interp))
+    # XXX: We should propagate edges down here too.
+    edges = Core.svec()
+    callee_result = structural_analysis!(callee_codeinst, Compiler.get_inference_world(interp), edges)
```

Comment:
Any suggestions on how to get edges down here? I don't see another way than to embed edges into `StructuralRefiner`, and I'd prefer to avoid having more intrusive changes in addition to those involving passing `edges` around in various functions.

Reply:
This should just have a single edge on the CodeInstance from the ADAnalyzer